authors/apps/profiles/migrations/0001_initial.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 2.1.2 on 2018-11-08 12:36
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
Why migrations?

Reply:
Migrations will reduce our build time on heroku. Moreover, It will avoid situations where the local db is not in sync with the remote. 
Pushing migrations is also recommended by Django docs